Gruntfile.js

```diff
@@ -1,13 +1,87 @@
 module.exports = function(grunt) {
-    // Project configuration
+
     'use strict';
-    grunt.loadNpmTasks('grunt-contrib-requirejs');
-    grunt.loadNpmTasks('grunt-filerev');
-    grunt.loadNpmTasks('grunt-regex-replace');
-    grunt.loadNpmTasks('grunt-karma');
-    grunt.loadNpmTasks('grunt-coveralls');
-    grunt.loadNpmTasks('grunt-jasmine-nodejs');
-    grunt.loadNpmTasks('grunt-contrib-uglify-es');
+
+    require('load-grunt-tasks')(grunt);
+    grunt.loadNpmTasks('@lodder/grunt-postcss');
+
+    // files required for the 'page.html' template
+    const kbaseStaticDir = 'kbase-extension/static/kbase/',
+
+    staticFilesNoConcat = [
+        kbaseStaticDir + 'custom/*.css',
+        kbaseStaticDir + 'css/*.css',
+        '!' + kbaseStaticDir + 'css/*_concat.css',
+    ],
+
+    pageCssFiles = [
+        "custom/custom.css",
+        "css/kbaseStylesheet.css",
+        "css/kbaseNarrative.css",
+        "css/kbaseIcons.css",
+        "css/landingPages.css",
+        "css/kbaseEditor.css",
+        "css/kbaseNotify.css",
+        "css/methodCell.css",
+        "css/bootstrapHelper.css",
+        "css/buttons.css",
+        "css/contigBrowserStyles.css", // may need to move this out
+        "css/kbaseJobLog.css",
+        "css/kbaseTour.css",
```

Comment:
Copied from `page.html`. The last three were not originally part of the set of css files loaded by `page.html`, but they do not contain selectors that conflict with those in the rest of the files, so loading them should be safe.